docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Device Identity
+description: >-
+  Learn about how WebAuthn is used to authenticate and authorize access using Device ID and state.
+sidebarDepth: 1
+---
+
+# Device Identity
+
+One of the core components of a zero-trust security model device identity, whereby a device's identity and state can be authenticated and factored into access control decisions. This topic page covers the concept of device identity and how it applies to the zero-trust model.
+
+## Why Device Identity Is Important
+
+The history of IT security has, until recently, mostly focused on user identity verification. In this model, access to a service is granted only after verifying a user's identity and authorization to that service. This was usually sufficient in scenarios where the user's access was physically limited to trusted devices in trusted physical and network spaces; a PC sitting in an office in the company headquarters on a local network, for example.
+
+Device identity is becoming more widely discussed as more products begin to use or require it. For example, Windows 11 generated a lot of news by [requiring TPM 2.0][win11-reqs], and Apple is taking steps to [eliminate passwords][apple-passkeys]:
+
+| ![Verge Article Header and Apple Video Page](./img/verge-apple.png) |
+|:--|
+| **Sources:**<br />- <https://www.theverge.com/2021/6/25/22550376/microsoft-windows-11-tpm-chips-requirement-security><br/>-  <https://developer.apple.com/videos/play/wwdc2021/10106/> |
+
+Device identity protects a trusted user from accessing sensitive data from a potentially unsafe device, like their personal computer or phone. Think of it as similar to multi-factor authentication (**MFA**); where MFA covers "what you know" (password) and "who you are" (biometrics, face recognition, etc), device identity asks "is this device safe?" by confirming that the device you are using to access a system is trusted.
+
+Device identity is similar but unique to MFA. Where MFA is an additional layer of protection on user identity, Device identity is a unique identifier that asserts trust of the hardware itself.
```

Comment:
Better, closer, warmer. How about:
```suggestion
Device identity is similar but unique to MFA. Where MFA is an additional layer of protection on user identity, hardware-backed device identity can be used to enhance and augment authorization decisions beyond the initial authentication flow. Where MFA increases the trust-level of a user, device identity provides trust for the device being used. More important, it denies access for non-trusted, potentially insecure devices.
```

Reply:
> More important, it denies access for non-trusted, potentially insecure devices.

I'm not sure if important needs to be an adverb; otherwise lgtm

Reply:
I've adectived the former adverb.
```suggestion
Device identity is similar but unique to MFA. Where MFA is an additional layer of protection on user identity, hardware-backed device identity can be used to enhance and augment authorization decisions beyond the initial authentication flow. Where MFA increases the trust-level of a user, device identity provides trust for the device being used. More importantly, it denies access for non-trusted, potentially insecure devices.
```